docs/modules/ROOT/pages/module/code.adoc

```diff
@@ -133,18 +133,44 @@ The following `defender.config.json` configuration file will disable Dependency
 }
 ----
 
-[[settings]]
-== Settings
+[[assets]]
+== Assets
+
+The Assets page allows you to manage your Code Inspector assets, supporting two types: Smart Contracts and GitHub repositories. When you install the Code Inspector GitHub app and select repositories, they will appear on this page. Additionally, you can create Smart Contract assets from addresses in your address book. Note that Smart Contract assets must be verified on Etherscan for this feature to work.
+
+Smart Contract assets can be manually triggered for reports on the report page. GitHub assets can be triggered manually and automatically from new pull requests (PRs) and commits.
+
+image::code-assets.png[Code Assets]
+
+[[asset-settings]]
+=== Settings
+
+The Asset Settings page offers two main settings to manage how Defender handles asset security: Automatically Generate Report and Vulnerability Detection.
+
+==== Automatically Generate Report
```

Comment:
I suggest being more descriptive in the title and using "Automatically Trigger Report for Pull Requests"